api/Window.json

```diff
@@ -3333,6 +3369,42 @@
               "deprecated": true
             }
           }
+        },
+        "relative-multi-screen": {
+          "__compat": {
+            "description": "Opened relative to the <a href='https://developer.mozilla.org/docs/Web/API/Window_Management_API/Multi-screen_origin'>Multi-screen origin</a>",
+            "support": {
+              "chrome": {
+                "version_added": "100"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1",
+                "partial_implementation": true,
+                "notes": "Behavior is inconsistent/unreliable."
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false,
+                "notes": "Opened relative to the origin of the primary screen."
```

Comment:
Safari 17's behavior here is closer to 'inconsistent' or 'unreliable'; it seems to depend on the relative arrangement of displays. For example:
- Screens arranged primary to the left/right of secondary - left|top are relative to primary
- Screens arranged primary above/below secondary - left|top are relative to the opener's current screen

It gets even weirder if the displays don't share a top or left edge, e.g.:
- Screens arranged primary diagonal below+left of secondary - left is relative to primary, but top is (erroneously clamped?) to the top of the primary screen.
- Screens arranged primary diagonal above+left of secondary - left is relative to primary, but top is (erroneously clamped?) to the bottom of the primary screen.